lib/ansible/modules/database/mongodb/mongodb_user.py

```diff
@@ -268,7 +271,10 @@ def user_remove(module, client, db_name, user):
         if module.check_mode:
             module.exit_json(changed=True, user=user)
         db = client[db_name]
-        db.remove_user(user)
+        if LooseVersion(PyMongoVersion) < LooseVersion('4.0'):
```

Comment:
```
if LooseVersion(PyMongoVersion) < LooseVersion('4.0'):
    db.remove_user(user)
db.command("dropUser", user)
```
You can avoid if/else in this case, improve code visibility

Reply:
Let me check if this would work for Mongo versions < 4.0